systems/WikibaseRepo/03-Context_and_Scope.md

```diff
@@ -2,4 +2,6 @@
 
 ### Business Context
 
+![Wikibase Repo business context diagram](./diagrams/03-business-context.drawio.svg)
+
```

Comment:
are we still missing the Neighbour/Description table here?

Reply:
done!